TSC-Charter.md

```diff
@@ -138,16 +138,29 @@ item has appeared to reach a consensus the moderator will ask "Does anyone
 object?" as a final call for dissent from the consensus.
 
 If an agenda item cannot reach a consensus a TSC member can call for
-either a closing vote or a vote to table the issue to the next meeting.
-The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue.
+either a closing vote or a vote to table the issue to the next meeting or to
+be conducted asynchronously through a system to be chosen by the TSC. The
+call for a vote is approved if none of the TSC members in the meeting object
+to it.
 
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the CPC.
 
+### Section 7.1. Immediate Votes
+
+For time-sensitive issues where reaching consensus asynchronously fails, TSC
+members may call for a vote through the TSC issue tracker. A compelling reason
```

Comment:
There have been a few occasions where the issue to vote on was private in nature. In those cases, we should use emails.